src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Patch.txt

```diff
@@ -54,3 +54,56 @@ Blank()
 
 >> Patch(t1, r1, {Field1:1/0}, {Field1:10});t1
 Table({Field1:10,Field2:"earth",Field3:DateTime(2022,1,1,0,0,0,0),Field4:true})
+
+>> Collect(t1,{Field1:3,Field2:"phobos",Field3:DateTime(2022,2,1,0,0,0,0),Field4:false});Collect(t1,{Field1:2,Field2:"deimos",Field3:DateTime(2022,2,1,0,0,0,0),Field4:false});Patch(t1,{Field2:"deimos"},{Field3:DateTime(2030,2,1,0,0,0,0)})
```

Comment:
>>> [](http://example.com/codeflow?start=0&length=2)

nit - I believe the .txt support multiline here, so we can leverage that for readability 
subsequent lines just need to be indented 2 spaces. 

And then the expected result is not indented.